configs/components/openssl-1.0.2.rb

```diff
@@ -61,6 +61,8 @@
       sslflags = '386'
     elsif platform.architecture =~ /aarch64$/
       target = 'linux-aarch64'
+    elsif platform.architecture =~ /ppc64le|ppc64el/ # Little-endian
+      target = 'linux-ppc64le'
```

Comment:
I don't think we should modify openssl 1.0.2 as we don't add support for 5.5.x stream